### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by tmampa**
- Refactored the project and replaced **JS objects** with **JS classes**
- The functionality of the app remains the same

# Update
- No more console error

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,70 +1,68 @@
...
-  removeBtn.innerText = 'Remove';
-  removeBtn.addEventListener('click', () => {
-    removeBook(div, book.title, book.author);
-  });
+    this.booksCollection.appendChild(div);
+  }
 
-  div.appendChild(removeBtn);
+  removeBook(element, title, author) {
+    element.remove();
```
Comment (GzimAsani) [inline index.js]: - [ ] Whenever I am trying to add another book with the same title I get an alert that the same title already exists and I cannot add it, that is fine but with it, I am getting an error in the console which you should fix it
Screenshot from 2021-11-25 19-24-06
.

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @tmampa  and @AlaaAlsalem                               
Good job so far! Your code looks fine, good job :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Great job with the project :muscle: 

## Required Changes ♻️ 
- [ ] _Check inline comments for more information :arrow_double_down:_ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (helman101) [reply index.js]: - [ ] I still see this  `console.log` error, kindly fix this before submitting another review request 😄
    imagen

**Review by helman101 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🔴 

Hi @tmampa & @AlaaAlsalem   👋, 

Nice job so far with this project 😄, but there are still some changes to do.

## Required Changes ♻️

- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

gif

If anything is not 100% clear please leave your questions or comments in the PR thread, we will be glad to help you 😄.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (tmampa) [reply index.js]: No more console error, thank you

**Review by Mihndim2020 — APPROVED** [review]

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @AlaaAlsalem, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**